### DISCUSSION TITLE
Accessing GraphQL schema files in a separate project

### DISCUSSION THREAD

**Discussion by debmitra** [Netflix/dgs-framework#217] [Q&A]
I have my GraphQL schema files in a project(ProjectA) and the application code which is in Java in a separate project(ProjectB). Both these projects are built using Apache Maven. My requirement is to load the schema files from ProjectA into ProjectB's classpath.

The DGS guide explains about having the application code and schema files inside same project where the schemas should be under src/main/resources/schema folder. However, there's no information on loading schemas and application code when they are in separate projects.

Please advise how to achieve this using DGS framework.

**Comment by paulbakker** [comment]
If you put a schema in the META-INF/schema folder of a JAR file, it will be picked up.

Note that there’s also work being done on a PR to make the schema path configurable: https://github.com/Netflix/dgs-framework/pull/210
For your use case the META-INF option seems the best fit though.
